sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_receipts_async.py

```diff
@@ -46,7 +46,7 @@ async def recognize_receipts(self):
         ) as form_recognizer_client:
 
             with open(path_to_sample_forms, "rb") as f:
-                poller = await form_recognizer_client.begin_recognize_receipts(receipt=f)
+                poller = await form_recognizer_client.begin_recognize_receipts(receipt=f, locale="en-US")
```

Comment:
also sorry for trampling on the reason of this PR, but is it necessary to include `locale` in the samples, esp since it's a kwarg and we're passing in the default value?

Reply:
Wanted to show how to pass it at least somewhere so added to one of the samples. I don't think it would make sense to use a non-english example, so thought explicitly passing it here wouldn't hurt.

Reply:
ok that's fair